holoviews/core/io.py

```diff
@@ -840,3 +841,101 @@ def contents(self, maxlen=70):
     def listing(self):
         "Return a list of filename entries currently in the archive"
         return ['.'.join([f,ext]) if ext else f for (f,ext) in self._files.keys()]
+
+
+
+class ArchiveReader(object):
+    """
+    ArchiveReaders abstract operations over different archive formats,
+    e.g. directory vs. zip file.
+    """
+    def __init__(self,archive):
+        self.archive_location = archive
+        self.root,self.archive_name=os.path.split(self.archive_location)
+
+    def open_member(self,name):
```

Comment:
Why not call this ``open_file``? As far as I can tell, all the ``ArchiveReaders`` contain files in some form or another and all these methods are concerned with files (even if they themselves are in a compressed archive). If this is true, I feel the generic word ``member`` obscures what is happening.